make.js

```diff
@@ -104,6 +104,7 @@ var COMMON_WEB_FILES =
       ['web/viewer.html'],
     COMMON_FIREFOX_FILES_PREPROCESS =
       [FIREFOX_CONTENT_DIR + 'PdfStreamConverter.jsm',
+       FIREFOX_CONTENT_DIR + 'PdfJsNetwork.jsm',
```

Comment:
Why is the `J` captalized here? Other files (e.g. the two below this line) do not have that.